sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -111,6 +111,7 @@
           <zeebe:header key="studentDataAdditionalTransportPlacement" value="$.data.additionalTransportPlacement" />
           <zeebe:header key="applicationStatus" value="$.applicationStatus" />
           <zeebe:header key="applicationHasNOAApproval" value="$.hasNOAApproval" />
+          <zeebe:header key="appealsStudentDependantsData" value="$.appeals.studentDependantsAppeal.submittedData.dependants[*][&#39;dateOfBirth&#39;,&#39;declaredOnTaxes&#39;,&#39;attendingPostSecondarySchool&#39;]" />
```

Comment:
appealsStudentDependantsAppealData ? following the pattern of other appeal variables.

Reply:
also please capture until the submittedData so that parent question `hasDependents` is also captured.